nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -39,6 +39,8 @@ WaypointFollower::WaypointFollower(const rclcpp::NodeOptions & options)
 
   declare_parameter("action_server_result_timeout", 900.0);
 
+  declare_parameter("global_frame_id", global_frame_id_);
```

Comment:
Lets actually put "map" in here, lets not make folks search for the first instance its set in the header